tests/test_text_encoding.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import unittest
+
+import torch
+from monai.networks.blocks.text_embedding import TextEncoder
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+
+
+class TestTextEncoder(unittest.TestCase):
+    def test_test_encoding_shape(self):
+        # test 2D encoder
+        text_encoder = TextEncoder(spatial_dims=2, out_channels=32, pretrained=True).to(device)
```

Comment:
please add the context manager to avoid reporting downloading issues, like this https://github.com/Project-MONAI/MONAI/blob/1a55ba5423d04d2ef7ac19356ccabc4c7906f577/tests/test_flexible_unet.py#L371

and remove the `print` commands as they are not needed. Then I can test and merge this soon, thanks!

Reply:
Thanks, removed.